README.md

```diff
@@ -39,8 +39,16 @@ And review the following guides:
 * [Configuring Remote Video Subscription](guides/configuring_remote_video_subscriptions.md)
 
 ## Include Amazon Chime SDK in Your Project
+You can download and integrate the Amazon Chime iOS SDKs to your project from both CocoaPods and binaries from Github release.
 
-To include the SDK binaries in your own project, follow these steps.
+### From CocoaPods
```

Comment:
If builder is using cocoapods, do they still need the second step of updating project file?
If not, "From CocoaPods" and "From Github Release Binaries" should be a level higher than "1 Download Binaries" and "2. Update Project File"